meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex_%.bbappend

```diff
@@ -1,4 +1,5 @@
-RPROVIDES_${PN}_mender-uboot += "u-boot"
+PROVIDES += "${@bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-uboot', 'u-boot', '', d)}"
+RPROVIDES_${PN} += "${@bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-uboot', 'u-boot', '', d)}"
```

Comment:
Indeed, it's not the first time I've seen the need for this. Something in the parsing prevents the suffix version from working correctly for `PROVIDES`.